catalog/dags/maintenance/add_license_url.py

```diff
@@ -227,12 +186,18 @@ def add_license_url():
         GROUP BY license, license_version
     """)
 
-    license_groups = get_license_groups(query)
-    updated = update_license_url.partial(batch_size="{{ params.batch_size }}").expand(
-        license_group=license_groups
-    )
-    report_completion(updated, query)
-    updated >> report_failed_license_pairs()
+    licenses = get_licenses(query)
+
+    trigger = TriggerDagRunOperator.partial(
+        task_id="trigger_batched_update",
+        trigger_dag_id=BATCHED_UPDATE_DAG_ID,
+        wait_for_completion=True,
+        execution_timeout=timedelta(hours=5),
+        max_active_tis_per_dag=1,
+        retries=0,
+    ).expand(conf=get_confs(licenses, batch_size="{{ params.batch_size }}"))
```

Comment:
Do you think we could do `map_index_template` here? Similar to this: https://github.com/WordPress/openverse/blob/bc457c020d4051d9d4e758fcfc90f635b8f9a9c2/catalog/dags/data_refresh/copy_data.py#L81

That way we could see the licenses as part of the mapped index/task name!

Reply:
I tried it in the previous DAG version as suggested in the [Airflow documentation](https://airflow.apache.org/docs/apache-airflow/stable/authoring-and-scheduling/dynamic-task-mapping.html#named-mapping), but an upstream issue prevented it from working with the parameters of the task: https://github.com/apache/airflow/issues/29366.

Do you see a workaround here?